android/app/src/main/java/org/uwaterloo/subletr/pages/home/map/HomeMapChildViewModel.kt

```diff
@@ -37,6 +37,7 @@ class HomeMapChildViewModel @Inject constructor(
 				}
 				else true
 			}
+			.skip(1)
```

Comment:
Why are we skipping one?

Reply:
Lol, that's a funny story.
So, if you don't move the slider before trying to select a listing item, then the app will interpret that as you moving the slider (because the `distinctOnChanged` only filters after the first item), so it'll refetch the item and deselect the item you selected after one second.